### ISSUE TITLE
All entities ignored when performing operations

### ISSUE THREAD

**Issue by ThePoptartCrpr** [boy0001/FastAsyncWorldedit#1092]
# Bug report

**Debug paste link**:

Couldn't get /fawe debugpaste to work, but here's the relevant log output at the time of the operation. Nothing out of the ordinary.
```
[11:12:20] [main/INFO]: [CHAT] §8(§4§lFAWE§8)§r§7 pos1 set to (-236, 5, -747) (32).
[11:12:21] [main/INFO]: [CHAT] §8(§4§lFAWE§8)§r§7 pos2 set to (-240, 4, -744) (40).
[11:12:23] [Server thread/WARN]: Keeping entity minecraft:cow that already exists with UUID 89ae6b15-ed81-4b41-bd9d-a1ebf34ac161
[11:12:23] [main/INFO]: [CHAT] §8(§4§lFAWE§8)§r§7 40 blocks affected
```
**Description of the problem:**

FAWE is advertised as fixing the annoying worldedit inconsistencies involving entities, but when performing any operation involving entities with FAWE installed, entities are fully ignored. This includes copying and pasting, moving, and stacking. In the above logs, the operation being performed was moving a selection with a cow. I've tried with keep-entities-in-blocks on both false and true, and the issue still persists.

**How to replicate**:

- Make any selection with an entity included (animal, item frame, etc)
- Attempt to perform an operation involving said selection
- Expect the entity to be modified with the operation
- Behold as the entity sits right there where it was

**Checklist**:

- [x] I included a `/fawe debugpaste` link
- [x] I made sure there aren't duplicates of this report [(Use Search)](https://github.com/boy0001/FastAsyncWorldedit/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of FAWE
- [x] I Made sure the bug/error is not caused by any other plugin

**Comment by ThePoptartCrpr** [comment]
Bump, has this been patched yet?

**Comment by SiuanSong** [comment]
Bumping again

**Comment by N0tMyFaultOG** [comment]
Your ticket has been closed because it falls into one of the following categories:

- Your issue has been fixed in a newer version of FastAsyncWorldEdit for versions 1.8 - 1.12.2, get the latest build [here](https://ci.athion.net/job/FastAsyncWorldEdit/lastSuccessfulBuild/artifact/target/) matching the platform you need.

- Your issue targets minecraft version 1.13 or higher. This is the wrong repository then. Move to [this repository](https://github.com/IntellectualSites/FastAsyncWorldEdit) and open a bug report here.
BUT BEWARE: You should consider updating to 1.15.2, using the latest Fawe build for 1.15.2 (Build 255 when writing this, get it [here](https://ci.athion.net/job/FastAsyncWorldEdit-1.15/)) and reproducing the issue in a clean environment (fresh setup Spigot or Paper server).

- Your issue has been fixed in a build of FAWE for 1.15 or an issue exists on our new repository.

- You either didn't fill out the template or deleted it.

If none of the bullet points suits, head to the second one.
